app/views/employee/index.html.erb

```diff
@@ -0,0 +1,64 @@
+<div class="main-container">
+  <div class="add-container">
+    <div class="add-button">
+      <span> + </span>
+    </div>
+    <div class="add-text-container">
+      <h1>Add new employee</h1>
+    </div>
+  </div>
+  <div class="table-container">
+    <section class="header-table">
+      <div>
+        <ul>
+          <li class="header-table-text">
+            <p>Email</p>
+          </li>
+          <li class="header-table-text">
+            <p>Name</p>
+          </li>
+          <li class="header-table-text">
+            <p>Position</p>
+          </li>
+          <li class="header-table-text">
+            <p>Number</p>
+          </li>
+          <li class="header-table-text">
+            <p>Private Number</p>
+          </li>
+          <li class="header-table-text">
+            <p>Actions</p>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <% @employees.each do |employee| %>
+      <section class="employees-table">
+        <ul class= "employees-table-list" >
+          <li class="eployees-table-cell">
+            <p><%= employee.email %></p>
+          </li>
+          <li class="eployees-table-cell">
+            <p><%= employee.name %></p>
+          </li>
+          <li class="eployees-table-cell">
+            <p><%= employee.position %></p>
+          </li>
+          <li class="eployees-table-cell">
+            <p><%= employee.employee_number %></p>
+          </li>
+          <li class="eployees-table-cell">
+            <p><%= employee.private_number %></p>
+          </li>
```

Comment:
All these classes are named how `eployees-table-cell` fix them for `employees-table-cell`